source/ed/connectors/fusio_parser.h

```diff
@@ -88,7 +88,7 @@ struct RouteFusioHandler : public GenericHandler {
     int ignored;
     void init(Data&);
     void handle_line(Data& data, const csv_row& line, bool is_first_line);
-    const std::vector<std::string> required_headers() const { return {"route_id", "route_name", "line_id"}; }
+    const std::vector<std::string> required_headers() const { return {"route_id", "route_name", "line_id", "is_forward"}; }
```

Comment:
line length: for readability we try to stay under 113 chars which is the limit in github